addons/website_sale/views/templates.xml

```diff
@@ -809,6 +809,8 @@
                     <input t-if="category" type="hidden" name="category" t-att-value="category.id" />
                     <input type="hidden" name="search" t-att-value="search" />
                     <input type="hidden" name="order" t-att-value="order"/>
+                    <input type="hidden" name="start_date" t-att-value="start_date"/>
+                    <input type="hidden" name="end_date" t-att-value="end_date"/>
```

Comment:
This won't solve the issue for existing clients as templates are only updated at upgrade. It will work for new customers though